src/finalfusion/vocab/subword.py

```diff
@@ -148,19 +148,16 @@ class FinalfusionBucketVocab(SubwordVocab):
     """
     def __init__(self,
                  words: List[str],
-                 indexer: FinalfusionHashIndexer = None):
+                 indexer: Optional[FinalfusionHashIndexer] = None):
```

Comment:
Missed that. I guess add-on typing is not really strong static typing.

Reply:
I saw it while writing testcases because PyCharm didn't highlight feeding a `FinalfusionHashIndexer` to `FastTextVocab`.

`mypy` docs actually state that defining a default arg as `None` will implicitly make the argument `Optional[]`. Didn't read that part until now but I guess being more verbose doesn't hurt here!